.gitignore

```diff
@@ -64,6 +64,7 @@ cover/
 local_settings.py
 db.sqlite3
 db.sqlite3-journal
+static_root/
```

Comment:
Probably better to move these out of the section that's autogenerated so we don't accidentally clobber them if we ever update the gitignore, as unlikely as that might be.